stable/appmesh-controller/README.md

```diff
@@ -62,8 +62,10 @@ aws iam create-policy \
 ```
 Take note of the policy ARN that is returned
 
-
 Create an IAM role for service account for the App Mesh Kubernetes controller, use the ARN from the step above
+
+> Note: if you deleted `serviceaccount` in the `appmesh-system` namespace, you will need to delete and re-create `iamserviceaccount`. `eksctl` does not override the `iamserviceaccount` correctly ([see this issue](https://github.com/weaveworks/eksctl/issues/2665))
```

Comment:
It would be less confusing if we could show how to delete existing iamserviceaccount.

Reply:
I did not add that on purpose as some users run commands copy-pasta style _without_ reading the comments so they may run the delete command without reading the reason why it needs to be run